include/librealsense2/h/rs_option.h

```diff
@@ -84,6 +84,7 @@ extern "C" {
         RS2_OPTION_LED_POWER, /**< Power of the LED (light emitting diode), with 0 meaning LED off*/
         RS2_OPTION_ZERO_ORDER_ENABLED, /**< Toggle Zero-Order mode */
         RS2_OPTION_ENABLE_MAP_PRESERVATION, /**< Preserve previous map when starting */
+        RS2_OPTION_FREEFALL_ENABLED, /**< Enable/disable sensor shutdown when a free-fall is detected (off by default) */
```

Comment:
It is not necessary a freefall - maybe we can consider a more generic term RS2_OPTION_~~FREEFALL_ENABLED~~**ANTI_SHOCK_PROTECTION** or similar

Reply:
I'd suggest FREEFALL_DETECTION_ENABLED